cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -3,7 +3,7 @@
 ## IMPORTANT PLEASE NOTE!
 Any time the file structure in the `windows` directory changes, `windows/BUILD`
 and `k8s.io/release/lib/releaselib.sh` must be manually updated with the
-changes. We HIGHLY recommend not changing the file structure, because consumers
+changes. We highly recommend not changing the file structure, because consumers
```

Comment:
i don't think this change is necessary.
please feel free to send more PRs!
/close